catalog-info.yaml

```diff
@@ -0,0 +1,58 @@
+# Declare a Backstage Component that represents the Rally application.
+---
+# yaml-language-server: $schema=https://json.schemastore.org/catalog-info.json
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: rally
+  description:  Macrobenchmarking framework for Elasticsearch
+  annotations:
+    backstage.io/source-location: url:https://github.com/elastic/rally/
+    github.com/project-slug: elastic/rally
+    github.com/team-slug: elastic/es-perf
+    buildkite.com/project-slug: elastic/rally
+  tags:
+    - elasticsearch
+    - benchmark
+    - python
+    - performance
+  links:
+    - title: Rally docs
+      url: https://esrally.readthedocs.io/en/stable/
+spec:
+  type: application
+  owner: group:es-perf
+  lifecycle: production
+  dependsOn:
+    - resource:marbi-buildkite-pipeline
```

Comment:
How does your service depends on marbi?

Reply:
My fault because I sent @pquentin to the Marbi catalog info in the absence of real documentation :)